### PR TITLE
Concat condition reason to it's type on condition union

### PR DESCRIPTION
**Pull request by jhadvig**
Per discussion with @deads2k 

/assign @deads2k

### CODE REVIEW

File: pkg/operator/status/condition.go [hunk 1]
```diff
@@ -95,6 +95,9 @@ func unionMessage(conditions []operatorv1.OperatorCondition) string {
 
 func unionReason(conditions []operatorv1.OperatorCondition) string {
 	if len(conditions) == 1 {
+		if len(conditions[0].Reason) != 0 {
+			return fmt.Sprintf("%s: %s", conditions[0].Type, conditions[0].Reason)
+		}
 		return conditions[0].Type
```
Comment (deads2k) [inline pkg/operator/status/condition.go]: `return conditions[0].Type+conditions[0].Reason` I htink

Comment (jhadvig) [reply pkg/operator/status/condition.go]: Thought we want it in the same format as https://github.com/openshift/library-go/blob/master/pkg/operator/status/condition.go#L90
Not sure if just joining Type and Reason without any additional punctuation right.

**Comment by jhadvig** [comment]
@deads2k PR updated

**Pull request merged**